release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -0,0 +1,851 @@
+available. This release uses Kubernetes 1.13. New features, changes, and known
+issues that pertain to {product-title} {product-version} are included in this
+topic.
+
+Red Hat did not publicly release {product-title} 4.0 and, instead, is releasing
+{product-title} {product-version} directly after version 3.11.
+
+{product-title} {product-version} clusters are available at
+https://cloud.openshift.com. The Unified Hybrid Cloud (UHC) application for
+{product-title} allows you to deploy OpenShift clusters to either on-premise or
+cloud environments.
+
+{product-title} {product-version} is supported on Red Hat Enterprise Linux 7.6 and
+later, as well as Red Hat Enterprise Linux CoreOS 4.1.
+
+You must use {op-system-first} for the control plane, or master, machines and
+can use either {op-system} or Red Hat Enterprise Linux 7.6 for compute, or worker,
+machines.
+
+[IMPORTANT]
+====
+Because only Red Hat Enterprise Linux version 7.6 is supported for compute
+machines, you must not upgrade the Red Hat Enterprise Linux compute machines to
+version 8.
+====
+
+You can install {product-title} {product-version} with installer-provisioned
+infrastructure on Amazon Web Services (AWS) or user-provided infrastructure on
+AWS, bare metal, or vSphere hosts. If you use the installer-provisioned
+infrastructure installation, the cluster provisions and manages all of the
+cluster infrastructure for you.
+
+{product-title} requires all machines, including the computer that you run the
+installation process on, to have direct internet access to pull images for
+platform containers and provide telemetry data to Red Hat. You cannot specify a
+proxy server for {product-title}.
+
+[[ocp-4-1-release-acknowledgements]]
+=== Acknowledgments
+
+Red Hat Global Support Services would like to recognize Rushil Sharma, JooHo
+Lee, and Suresh Gaikwad for their outstanding contributions in evaluating and
+testing {product-title} 4.1.
+
+[[ocp-41-deprecated-features]]
+=== Deprecated features
+
+Large changes to the underlying architecture and installation process are
+applied in {product-title} 4.1, and many features from {product-title} 3.x are
+now deprecated.
+
+.Features Deprecated in Version 4.1
+[cols="2",options="header"]
+|====
+|Feature |Justification
+
+|Hawkular
+|Replaced by cluster monitoring.
+
+|Cassandra
+|Replaced by cluster monitoring.
+
+|Heapster
+|Replaced by Prometheus adapter.
+
+|Atomic Host
+|Replaced by Red Hat CoreOS.
+
+|System containers
+|Replaced by Red Hat CoreOS.
+
+|projectatomic/docker-1.13 additional search registries
+|CRI-O is the default container runtime for 4.x on RHCOS and Red Hat Enterprise Linux.
+
+|`oc adm diagnostics`
+|Operator-based diagnostics.
+
+|`oc adm registry`
+|Replaced by the registry operator.
+
+|Custom strategy builds using Docker
+|If you want to continue using custom builds, you should replace your Docker
+invocations with Podman and Buildah. The custom build strategy will not be
```

Comment:
```suggestion
invocations with Podman or Buildah. The custom build strategy will not be
```

`podman build` is a drop-in replacement for `docker build`. You can use a Dockerfile with buildah via `buildah bud ...`, but buildah lets you do a whole lot more.
/cc @rhatdan